backend/compact-connect/lambdas/python/provider-data-v1/handlers/registration.py

```
@@ -230,13 +230,108 @@ def register_provider(event: dict, context: LambdaContext):  # noqa: ARG001 unus
+                            logger.info(
+                                'Deleted old Cognito user for re-registration',
+                                old_email=provider_record.compactConnectRegisteredEmailAddress
+                            )
+                        except ClientError as delete_e:
+                            logger.error(
+                                'Failed to delete old Cognito user during re-registration',
+                                error=str(delete_e),
+                                old_email=provider_record.compactConnectRegisteredEmailAddress
+                            )
+                            # Continue with registration anyway, as the new email might still work
+                    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)